TeamCode/src/main/java/org/firstinspires/ftc/teamcode/rework/RobotTools/Movements.java

```diff
@@ -208,7 +209,16 @@ private boolean isDone(ArrayList<Waypoint> path, Point center, double heading) {
         return (Math.hypot(center.x - endPoint.x, center.y - endPoint.y) < distanceThreshold) && (!willAngleLock || Math.abs(angleWrap2(angleLockHeading - heading)) < angleThreshold) && pathIndex == path.size() - 2;
     }
 
-    public boolean isFileDump(){
+    public boolean isFileDump() {
         return isFileDump;
     }
+
+    @Override
+    public ArrayList<String> getTelemetryData() {
```

Comment:
Is it important to synchronize this?

Reply:
no it should be fine; gettelemetrydata shouldn't have any side effects in it